README.md

```diff
@@ -423,24 +411,31 @@ To solve this error You must go to [hoyolab.com](https://www.hoyolab.com/genshin
 Every endpoint in mihoyo's api requires authentication, this is in the form of a cookie and an authkey.
 User stats use a cookie and wish history uses an authkey.
 
-The cookie is bound to the user and as far as I know can only be reset by changing your password, so remember to never give your cookie to anyone. For extra safety you may want to create an alt account, so your real account is never in any danger. This token will allow you to view public stats of all users and private stats of yourself.
+The cookie is bound to the user and as far as I know, it can only be reset by changing your password. Safety wise, these cookies only give an access to your own statistics and allow few actions (claim daily reward) so it isn't dangerous by itself, but you should still keep it safe, as Mihoyo's API could include more sensitive features in the future. For extra safety you may want to create an alt account, so your real account is never in any danger. This alt token will allow you to view public stats of all users and private stats of yourself.
```

Comment:
*only* give access to your own statistics? You're definitely making it seem way too safe.

Reply:
Hmm, that's right: depends on _which_ cookie you're having access to, I'll make it clearer

Reply:
I'd rather avoid explicitly saying that's it's not dangerous cause it absolutely is. I'm fine with being overly cautious but I'm totally against saying there are no problems at all because it indeed is dangerous by itself.